libraries/AP_HAL_ChibiOS/hwdef/CubeOrangePlus/hwdef.dat

```diff
@@ -22,14 +22,43 @@ APJ_BOARD_ID 1063
 
 undef CHECK_ICM20602_EXT
 undef CHECK_IMU0_PRESENT
+undef CHECK_IMU1_PRESENT
+undef CHECK_IMU2_PRESENT
+undef CHECK_ICM20649
+undef BOARD_VALIDATE
 undef IMU
 
+PG1 ICM45686_CS CS
+
+#IMU 0
 SPIDEV icm42688_ext   SPI4 DEVID4  ACCEL_EXT_CS  MODE3  2*MHZ  8*MHZ
 
-IMU Invensensev3 SPI:icm42688_ext ROTATION_YAW_90
-IMU Invensensev2 SPI:icm20948_ext ROTATION_PITCH_180
-IMU Invensensev2 SPI:icm20948 ROTATION_YAW_270
+#IMU 1
+SPIDEV icm42688_ext2  SPI4 DEVID4  GYRO_EXT_CS   MODE3  2*MHZ  8*MHZ
```

Comment:
The 42688 (and I suspect the 45686) is rated to 24Mhz SPI - its worth checking if you can go higher than 8Mhz as this significantly reduces the loop time

Reply:
Yes, they are. But we have been using 8 MHz bus on ICM42688, pushing to 24MHz is bit of a risk. The intention is to potentially backport some of this.

Reply:
right, but even at 12Mhz you get a significant win - this seemed to be the sweet spot that AP allowed. I could not get 24Mhz on other boards because of the clock tree constraints.